src/components/ResourceBrowser/ResourceBrowser.service.tsx

```diff
@@ -41,8 +58,31 @@ export const getResourceGroupList = (clusterId: string, signal?: AbortSignal): P
 export const createNewResource = (resourceListPayload: CreateResourcePayload): Promise<CreateResourceResponse> =>
     post(Routes.K8S_RESOURCE_CREATE, resourceListPayload)
 
-export const deleteResource = (resourceListPayload: ResourceListPayloadType): Promise<CreateResourceResponse> =>
-    post(Routes.DELETE_RESOURCE, resourceListPayload)
+// export const deleteResource = (
```

Comment:
Please ignore this & the one commented out for `deleteResource`. Committed and pushed the mock here so that I can deploy it on ent-7 and get it UAT'd with out actually hitting any delete or restart apis